lighthouse-core/lib/i18n/i18n.js

```diff
@@ -315,4 +319,5 @@ module.exports = {
   createMessageInstanceIdFn,
   getFormatted,
   replaceIcuMessageInstanceIds,
+  MESSAGE_INSTANCE_ID_REGEX,
```

Comment:
add comment that it's exported only for testing?

Reply:
> add comment that it's exported only for testing?

I added an `isIcuMessage()` instead, just because we do the same check a couple of times internally, so nice in there and nicely encapsulated externally